buildenv/jenkins/docker-slaves/x86/centos6.9/Dockerfile

```diff
@@ -263,19 +263,26 @@ RUN mkdir /home/${USER}/openjdk_cache \
   && git remote add omr https://github.com/eclipse/openj9-omr.git \
   && git fetch --all
 
-# Install OpenSSL v1.1.0
+# Install OpenSSL v1.1.1b
+# Required for JITaaS
 RUN cd /tmp \
-  && wget https://www.openssl.org/source/old/1.1.0/openssl-1.1.0g.tar.gz \
-  && tar -xzf openssl-1.1.0g.tar.gz \
-  && rm -f openssl-1.1.0g.tar.gz \
-  && cd /tmp/openssl-1.1.0g \
-  && ./config -Wl,-rpath,/usr/local/lib64 \
+  && wget https://github.com/openssl/openssl/archive/OpenSSL_1_1_1b.tar.gz \
+  && tar -xzf OpenSSL_1_1_1b.tar.gz \
+  && rm -f OpenSSL_1_1_1b.tar.gz \
+  && cd /tmp/openssl-OpenSSL_1_1_1b \
+  && ./config --prefix=/usr/local/openssl-1.1.1b --openssldir=/usr/local/openssl-1.1.1b \
   && make \
   && make install \
   && cd .. \
-  && rm -rf openssl-1.1.0g
+  && rm -rf openssl-OpenSSL_1_1_1b \
+  && echo "/usr/local/openssl-1.1.1b/lib" > /etc/ld.so.conf.d/openssl-1.1.1b.conf
+ENV PATH="/usr/local/openssl-1.1.1b/bin:${PATH}"
+# Configure SSH to use new openssl
+RUN echo 'PATH=/usr/local/openssl-1.1.1b/bin:$PATH' > /home/${USER}/.ssh/environment \
```

Comment:
This line will cause the path to become: `/usr/local/openssl-1.1.1b/bin:$PATH:/usr/local/sbin:/usr/sbin:/sbin:/home/jenkins/bin`
Because of the stuff that was removed from the path the ssh-agent binary couldn't be found which is why the compiles were failing.
I changed this line to declare an exact path, built my own image, and tested it, and it [passed.](https://hyc-runtimes-jenkins.swg-devops.com/view/OpenJ9%20-%20Build/job/Build_JDK8_x86-64_linux_Personal/140/)